src/company_cmd.cpp

```diff
@@ -111,8 +111,11 @@ void SetLocalCompany(CompanyID new_company)
 
 	_current_company = _local_company = new_company;
 
-	/* Delete any construction windows... */
-	DeleteConstructionWindows();
+	/* Recreate company windows */
+	if (Company::IsValidID(new_company))
+	{
```

Comment:
The opening brace of if, for, while etc. statements needs to go on the same line as the statement. Or make a braces-free single-statement body on one line.